src/Phinx/Migration/AbstractMigration.php

```diff
@@ -82,17 +82,6 @@ abstract class AbstractMigration implements MigrationInterface
         if ($output !== null) {
             $this->setOutput($output);
         }
-
-        $this->init();
-    }
-
-    /**
-     * Initialize method.
-     *
-     * @return void
-     */
-    protected function init()
```

Comment:
Why are you removing the method from the abstract class and then needing method_exists() check?
Couldnt we keep this method here?

Reply:
I can either remove it here or add a `public function int()` to the AbstractInterface.php and change this from `protected` to `public` here (for both seed and migration). I'm open to either, but was following the convention of up/down/change not existing (and since `run()` in AbstractSeed was protected by a `function_exists()` call).

Reply:
I see. Do others have any opinion here?